packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -1017,11 +1017,15 @@ class _RenderRangeSlider extends RenderBox {
   void _handleDragUpdate(DragUpdateDetails details) {
     final double dragValue = _getValueFromGlobalPosition(details.globalPosition);
 
-    // If no selection has been made yet, use the velocity of the drag to
-    // determine which thumb should be selected.
+    // If no selection has been made yet, test for thumb selection again now
+    // that the value if dx is non zero. If this is the first selection of the
```

Comment:
> `now that the value if dx is non zero`

if/of typo? And what is dx?

Reply:
Typo fixed. dx is the x displacement of the drag offset. Velocity was used to refer to dx, but technically it is a displacement so that has been updated